docs/misc/vmdkops-admin-cli-spec.md

```diff
@@ -121,6 +121,18 @@ Show any interesting information about the service. This includes file paths of
 information, and PID of running service. A simple example is shown here, although it's possible
 that the exact format may be somewhat different.
 
+# Set
+Modify attribute settings on a given volume. The volume is identified by its full path on the datastore,
+for example if the volume name is `container-vol` then the volume is specified as "/vmfs/volumes/<datastore>/dockvols/container-vol.vmdk".
+The attributes to set/modify are specified as a comma separated list as "<attr1>=<value>, <attr2>=<value>....". For example,
+a command line would look like this.
+
+```
+$ vmdkops-admin set --volume=<full path of volume> --options="<attr1>=<value>, <attr2>=<value>, ..."
```

Comment:
The issue is the plugin gets to know the VM and can figure which dockvols
folder to create the volume in. But the user on the ESX always needs to
know the datastore where the volume is located. Otherwise we can't locate
the volume at all. Hence the need to give the path.

Alternative is for the user to always give volume@datastore as the format.
I can change to make it that way.

On Fri, Aug 26, 2016 at 9:18 PM, Prashant Dhamdhere <
notifications@github.com> wrote:

> In docs/misc/vmdkops-admin-cli-spec.md
> <https://github.com/vmware/docker-volume-vsphere/pull/583#discussion_r76441304>
> :
>
> > @@ -121,6 +121,18 @@ Show any interesting information about the service. This includes file paths of
> >  information, and PID of running service. A simple example is shown here, although it's possible
> >  that the exact format may be somewhat different.
> >
> > +# Set
> > +Modify attribute settings on a given volume. The volume is identified by its full path on the datastore,
> > +for example if the volume name is `container-vol` then the volume is specified as "/vmfs/volumes/<datastore>/dockvols/container-vol.vmdk".
> > +The attributes to set/modify are specified as a comma separated list as "<attr1>=<value>, <attr2>=<value>....". For example,
> > +a command line would look like this.
> > +
> > +```
> > +$ vmdkops-admin set --volume=<full path of volume> --options="<attr1>=<value>, <attr2>=<value>, ..."
>
> Should we be using volumeName@datastore similar to what we use in docker
> run command. I don't think vmdkops-admin prints full volume path in any
> command.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/vmware/docker-volume-vsphere/pull/583/files/47e499c2476535f28ed5fb4c4bae13ad40ddd580#r76441304>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/APHseNms_WatJyPvR93TwJgEy8Zkc7l2ks5qjwrdgaJpZM4Jpr3h>
> .
>